frontend/public/components/utils/simple-tab-nav.tsx

```diff
@@ -31,9 +35,9 @@ export const SimpleTabNav: React.FC<SimpleTabNavProps> = ({
     <div>
       <Tabs
         onSelect={handleTabClick}
-        className={additionalClassNames}
+        className={classnames({ 'pf-v6-u-mb-lg': withinSidebar }, additionalClassNames)}
         defaultActiveKey={selectedTab || tabs[0]?.name}
-        inset={{ default: 'insetNone', xl: 'insetSm' }}
+        inset={!withinSidebar && { default: 'insetNone', xl: 'insetSm' }}
```

Comment:
I'm thinking it's better for the inset not to change (one less difference).

```suggestion
        inset={!withinSidebar && { default: 'insetNone' }
```